softwareComponents/configuration/include/configuration/rofibot.hpp

```diff
@@ -39,6 +39,7 @@ namespace roficom {
      * Return a corresponding angle in radians for a given orientation
      */
     double orientationToAngle( Orientation o = Orientation::North );
+    Matrix orientationToTransform(roficom::Orientation orientation);
```

Comment:
Missing spaces around the argument.

Reply:
Addressed in [4eb8bdd...1ac00c2](https://github.com/paradise-fi/RoFI/compare/4eb8bddc3e832cfe0224124baa72ddffc874d582..1ac00c2ee26269312e7b13afced9d5cfe4f22cda)